src/test/java/com/hocs/test/pages/foi/Approval.java

```diff
@@ -38,15 +38,15 @@ public void selectApprovalStatus(String status) {
     }
 
     public void selectApprovalRequestDecision() {
-        recordCaseData.selectRandomRadioButtonFromGroupWithHeading("Decision");
+        recordCaseData.selectSpecificRadioButtonFromGroupWithHeading("Approve","Decision");
```

Comment:
Is there a difference in outcome selecting 'Approve' or 'Reject'? If it doesn't impact the displayed request status i.e Due, Overdue, Completed, Cancelled etc. then I think we should keep the selection random

Reply:
Yeah, there is a difference in outcome if we choose Reject then the case wont go further as all the approvals needs to be approved

Reply:
I've double checked (in Delta), and specifying either Approve or Reject works the same way. Is there potentially some confusion between Decision and Approval Status?

Reply:
The new functionality is in gamma and will deploy the code
 to delta in couple of days as we are about to finish few more stories for PM review

Reply:
Ok no problem